cluster_config.go

```diff
@@ -519,6 +519,13 @@ func (cfg *Config) GetPeerstorePath() string {
 	return filepath.Join(cfg.BaseDir, filename)
 }
 
+func (cfg *Config) String() string {
+	jcfg, _ := cfg.toConfigJSON()
+
+	hidden := []string{"ID", "PrivateKey", "Secret"}
```

Comment:
ID is not secret

Reply:
ID is not part of config. It has been moved to `identity.go`. ID is part of config only for compatibility,